insights/parsers/pmrep.py

```diff
@@ -54,7 +54,7 @@ def search(self, **kwargs):
 
         Examples:
             >>> pmrep_doc_obj.search(name__endswith='lo')
-            [{'name': 'network.interface.out.packets-lo', 'value': '1.000'}, {'name': 'network.interface.collisions-lo', 'value': '3.000'}]
+            [{'name': 'network.interface.collisions-lo', 'value': '3.000'}, {'name': 'network.interface.out.packets-lo', 'value': '1.000'}]
```

Comment:
Since the py2 and py3 have different sort policies, you'd better re-sort it in the test. See L16

Reply:
@xiangce - Updated in line 61, please have a check.